src/index.html

```diff
@@ -6,7 +6,327 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo__logo"
+          class="header__logo-pic"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
```

Comment:
add empty lines between multiline sibling blocks of HTML
![image](https://user-images.githubusercontent.com/22824431/224998312-9217c9c0-3be3-4fa2-b53f-b37034d5240b.png)
```suggestion
          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
          
          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
```